spread-tests/install-go.sh

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+set -e
+
+apt-get update
+apt-get install golang -y
```

Comment:
The snap go install is the way forward I think. 

```sudo snap install go --channel=1.14```

Reply:
Would maybe be great if we can install the minimum version from ```go.mod```

Reply:
Michael argued that using a vague installation process for Go here would inherently test that pebble stays compatible with newer versions of Go. I guess that's an advantage too, but indeed I'd rather control that dependency. I'll make the change.